packages/twenty-server/src/engine/api/graphql/graphql-query-runner/decorators/on-database-event.decorator.ts

```diff
@@ -0,0 +1,14 @@
+import { OnEvent } from '@nestjs/event-emitter';
+
+import { DatabaseEventAction } from 'src/engine/api/graphql/graphql-query-runner/services/api-event-emitter.service';
+
+export function OnDatabaseEvent(
+  object: string,
+  action: DatabaseEventAction,
+): MethodDecorator {
+  const event = `${object}.${action}`;
+
+  return (target: any, propertyKey: string, descriptor: PropertyDescriptor) => {
```

Comment:
I guess we can't do better than any here? It's completely fine with me as this typing is internal

Reply:
looking at the internal typing of event-emitter package, Object is more accurate

Reply:
ok